README.md

```diff
@@ -46,6 +46,8 @@ $ octopilot \
     --repo "my-org/another-repo(merge=true)" \
     --repo "discover-from(env=PROMOTE_TO_REPOSITORIES)" \
     --repo "discover-from(query=org:my-org topic:my-topic)" \
+    --repo "discover-from(searchtype=code,query=org:my-org filename:my-file path:dir-path in-file-text)" \
+    --repo "discover-from(searchtype=code,query=org:my-org filename:my-file path:dir-path fork:true)" \
```

Comment:
what about doing:

```
discover-from(query=org:my-org topic:my-topic)
```

for a classic repo search

and

```
discover-from(codequery=org:my-org filename:my-file path:dir-path in-file-text)
```

for a github code search?

Reply:
I feel it's easier to manage a single query variable name without having to handle all possible xxxquery name. And with searchType then we can just precise what kind of search we want (repositories by default or code if we want code search).

But i'm still open to change if you feel it better